src/entities/Proposal/discourse-proposal-update-message-builder.ts

```diff
@@ -0,0 +1,58 @@
+import { ProposalAttributes, ProposalStatus } from "./types"
+import { Vote } from "../Votes/types"
+import { calculateResult } from "../Votes/utils"
+
+export class DiscourseProposalUpdateMessageBuilder {
```

Comment:
A shorter file name would be great. Also does this need to be an object?
Maybe a utility function in /entities/Proposal/utils.ts should be enough.